blocks/header/header.css

```diff
@@ -175,10 +175,20 @@ header nav .nav-sections ul > li > a {
 
 header nav .nav-sections ul > li > ul {
   margin-top: 0;
+  width: 300px;
+  float: left;
+  display: block;
+}
+
+ul li.nav-drop > ul > li {
+    font-weight: bold !important;
+    margin-bottom: 2em;
 }
 
 header nav .nav-sections ul > li > ul > li {
   font-weight: 400;
+    width: 300px;
```

Comment:
are we sure that it should be 300px? or should we use a variable/configuration?

Reply:
Changed hardcoded values for CSS variables